### DISCUSSION TITLE
Filtres établissement

### DISCUSSION THREAD

**Discussion by LHuberson** [unistra/esup-immersup-demandes#17] [6- Administration]
Est -il possible d'avoir un système de filtre des établissements? 
Par exemple Sarthe / Mayenne, Privé / Public, Lycées pilotes pour des PIA, Etablissement encordé...
=> Ce filtre pourrait t'il être applicable lorsque l'on crée un évènement et que l'on souhaite le restreindre à un type d'établissement?

**Comment by efinkbeiner5** [comment]
Doublon avec la demande #41

**Comment by efinkbeiner5** [comment]
On continue à suivre le sujet dans la demande #41
